BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/SearchEngineView/DropDownSearchEngineView.swift

```diff
@@ -41,7 +41,7 @@ final class DropDownSearchEngineView: UIView, SearchEngineView, ThemeApplicable
         fatalError("init(coder:) has not been implemented")
     }
 
-    func configure(_ config: LocationViewConfiguration, delegate: LocationViewDelegate) {
+    func configure(_ config: LocationViewConfiguration, _ isLocationTextCentered: Bool, delegate: LocationViewDelegate) {
```

Comment:
Nit could we remove `_` for boolean i think it helps using the method, knowing the parameter name: 
```suggestion
    func configure(_ config: LocationViewConfiguration, isLocationTextCentered: Bool, delegate: LocationViewDelegate) {
```